.github/ISSUE_TEMPLATE/joinin.md

```diff
@@ -0,0 +1,23 @@
+Welcome to OpenLab!
+
+Here are some ways to support OpenLab:
+
+* Additional infrastructure (commodity hardware, cloud resources, etc)
+* Engineering support (developers, project management, software support)
+* QA, QE, and test (unit, integration, e2e, scenario) builders.
+* Software/application and tooling integrations
+* Operations support
+
+If you would like to join us, please leave your contact information below, we will invite you into OpenLab Team later. As a member you will find out what's happening in OpenLab, what fellow open source community members are doing using OpenLab resources, also you can access OpenLab resources for your own testing.
+
+### Full Name
+
+### Email
+
+### Organization
```

Comment:
1. Github account id
2. IRC nickname (Optional)
3. Company/Organization
4. Job Title (Optional)
5. Phone (Optional)

Reply:
I think email is needed.

Reply:
Yes, it is